tests/Unit/Evolution/Systems/GrMhd/GhValenciaDivClean/Test_Characteristics.cpp

```diff
@@ -0,0 +1,42 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <limits>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/Characteristics.hpp"
+#include "Evolution/Systems/GrMhd/GhValenciaDivClean/Characteristics.hpp"
+#include "Framework/TestHelpers.hpp"
+#include "Helpers/PointwiseFunctions/GeneralRelativity/TestHelpers.hpp"
+
+namespace {
+void check_max_char_speed(const DataVector& used_for_size) {
+  MAKE_GENERATOR(gen);
+
+  const Scalar<DataVector> gamma_1{used_for_size.size(), 0.0};
```

Comment:
LGTM. Just a quick question: do we need to test with `gamma_1` being other than 0.0 here? Like some random value between -1 and 0, as the doc says.

Reply:
That's just guaranteed analytically, so would only be testing that the GH implementation is correct...